app/models/user.rb

```diff
@@ -1,6 +1,18 @@
 class User < ApplicationRecord
   has_many :votes
   has_many :ranked_works, through: :votes, source: :work
-
+  
   validates :username, uniqueness: true, presence: true
```

Comment:
This line prevents login since `username` is not a field that we're getting from github. Once commented out, login works.